### PR TITLE
refactor(tests): convert a subset of enzyme tests to use react testing library

### PR DESCRIPTION
**Pull request by KaylaBrady**
A subset of files from https://app.asana.com/0/1152340551558956/1201708616021563/f.

### CODE REVIEW

File: assets/tests/components/app.test.tsx [hunk 1]
```diff
@@ -37,7 +36,9 @@ describe("App", () => {
         <App />
       </SocketProvider>
     )
-    expect(wrapper.exists(".c-modal")).toBeTruthy()
+    expect(
+      result.queryByText(/Your connection to Skate has expired./)
+    ).not.toBeNull()
```
Comment (KaylaBrady) [inline assets/tests/components/app.test.tsx:41]: I used `.not.toBeNull` here for consistency with other tests. I've previously seen `toBeVisible()` used from [jest-dom](https://github.com/testing-library/jest-dom#tobevisible), which I found helpful for brevity when first learning react testing, but I don't feel strongly enough about it to break the existing testing conventions here!

Comment (lemald) [reply assets/tests/components/app.test.tsx:41]: I actually wasn't aware of `toBeVisible`, that actually looks better in at least some respects (its assertions cover more cases that might lead to the element not showing up on the page, and the naming is more clear about what we're looking to test). Let's use it going forward!

**Comment by lemald** [comment]
@KaylaBrady I see this is still marked as a draft, so I'll hold off on approving it until you mark it as ready for review, but this is looking really good so far!

**Review by lemald — APPROVED** [review]
👍

**Pull request merged**